tests/integration_tests/modules/test_user_events.py

```diff
@@ -77,7 +75,7 @@ def _test_network_config_applied_on_reboot(client: IntegrationInstance):
     assert 'dummy0' not in client.execute('ls /sys/class/net')
 
     _add_dummy_bridge_to_netplan(client)
-    client.execute('rm /var/log/cloud-init.log')
+    client.execute('echo "" > /var/log/cloud-init.log')
```

Comment:
Is this just in case we haven't yet run cloud-init at all so that _test_network_config_applied_on_reboot doesn't fail?

Don't we want both rm /var/log/cloud-init.log and then echo "" > /var/log/cloud-init.log so that we don't accidentally match "old boot" logs?

Reply:
`>` essentially does that, right?

Reply:
HA!. yes sorry, right.

Reply:
I changed the line because I was occasionally getting failures from `/var/log/cloud-init.log` not existing, which is really strange. I changed it to empty out the file completely, but then I didn't get any errors anymore. `restart()` has a built in wait, so the cloud-init log should be 'full' by the time we try to read it. I'm curious if adding a `sync` to Azure at the end of `wait` would fix it.